src/storage/storage.go

```diff
@@ -3,15 +3,17 @@ package storage
 import (
 	"database/sql"
 	"fmt"
+	"log"
 	"os"
 	"time"
 
 	"github.com/0xb10c/bademeister-go/src/types"
 	_ "github.com/mattn/go-sqlite3"
 )
 
-const VERSION = 2
+const currentVersion = 4
```

Comment:
IMO it is fine to bump more than 1 version per PR but not required

Reply:
Agree for future PRs, won't change it back now..